apps/backend/src/db/migrations/20250818155648_add_new_fields_to_meeting_status.ts

```diff
@@ -0,0 +1,56 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	MEETINGS: "meetings",
+} as const;
+
+const ColumnName = {
+	STATUS: "status",
+} as const;
+
+const enumName = "meeting_status";
+
+const MeetingStatus = {
+	ENDED: "ended",
+	FAILED: "failed",
+	JOINING: "joining",
+	RECORDING: "recording",
+	STARTED: "started",
+} as const;
+
+async function down(knex: Knex): Promise<void> {
+	await knex.transaction(async (trx) => {
+		await trx(TableName.MEETINGS)
+			.whereIn(ColumnName.STATUS, [
+				MeetingStatus.FAILED,
+				MeetingStatus.JOINING,
+				MeetingStatus.RECORDING,
+			])
+			.update({ [ColumnName.STATUS]: MeetingStatus.STARTED });
```

Comment:
If the status is FAILED, we should change it to ENDED

Reply:
No, if the status is FAILED, then when undoing the migration the status should become ENDED. In your case in becomes STARTED